Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -141,7 +141,8 @@ def list_incidents(self, filters, limit):
             url_suffix='/alerts/',
             json_data={
                 'filters': filters,
-                'limit': limit
+                'limit': limit,
+                'sortDirection': 'asc'
```

Comment:
Does that fix the issue because it makes sure the first incident is the oldest one?

Reply:
It turns out that without the "sortDirection" the first incident was the newest one.
Therefore the last fetch was not updated because after the first message all the messages were older and did not change it.